tests/bedrock/test_bedrock_service.py

```diff
@@ -2,18 +2,19 @@
 from pytest_mock import MockerFixture
 
 from app.bedrock import models, service
-from tests.fixtures import bedrock as bedrock_fixtures
```

Comment:
I've changed these bedrock test helpers to proper fixtures so we don't need to explicitly import them.